tin-drink/src/components/content/drinkList/SingleDrink.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function SingleDrink(props) {
+
+    console.log(props);
+    return(
+        <div id="single-drink" className="single-drink">
+        <div id="drink-photo" className="drink-photo">
+            <img src={props.photoPath}/>
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.